test/Tests/Writers/HTML.hs

```diff
@@ -127,10 +127,9 @@ tests =
           , "<p>Some more text.</p>"
           , "<div class=\"footnotes footnotes-end-of-document\">"
           , "<hr />"
-          , "<ol>"
-          , "<li id=\"fn1\"><p>Down here.<a href=\"#fnref1\" class=\"footnote-back\">↩︎</a></p></li>"
-          , "<li id=\"fn2\"><p>The second note.<a href=\"#fnref2\" class=\"footnote-back\">↩︎</a></p></li>"
-          , "</ol>"
+          , ""
+          , "<div id=\"fn1\"><p><a href=\"#fnref1\" class=\"footnote-back\">1.</a> Down here.</p></div>"
+          , "<div id=\"fn2\"><p><a href=\"#fnref2\" class=\"footnote-back\">2.</a> The second note.</p></div>"
```

Comment:
I don't think I like this change.  Using ordered lists gives us much better formatting (content of a note is aligned), and it will work well even if the note is, say, a code block.